classes/PodsInit.php

```diff
@@ -1756,10 +1842,42 @@ public function run() {
 		// Compatibility for Query Monitor conditionals
 		add_filter( 'query_monitor_conditionals', array( $this, 'filter_query_monitor_conditionals' ) );
 
-		// Add WP-CLI commands
+		// Remove Common menus
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 4baa4f4b9f1d7680a6e3d463a595b252</sub>